devel/ci/integration/tests/test_bodhi_tasks.py

```diff
@@ -113,6 +113,7 @@ def find_update():
         "bodhi",
         "updates",
         "request",
+        "--debug",
```

Comment:
Do we want to keep this ? Or was it needed only during development ?

Reply:
Yeah I think it can be useful, to give us more info whenever the test fails in the future.